apps/meteor/ee/server/lib/ldap/Manager.ts

```diff
@@ -43,7 +46,15 @@ export class LDAPEEManager extends LDAPManager {
 				await this.updateExistingUsers(ldap, converter);
 			}
 
+			const membersOfGroupFilter = await ldap.searchMembersOfGroupFilter();
+
 			await converter.convertUsers({
+				beforeImportFn: (async (data: IImportUser): Promise<boolean> => {
+					if (!ldap.options.groupFilterEnabled) {
+						return true;
+					}
+					return membersOfGroupFilter.includes(data.importIds[0]);
```

Comment:
You need to use the value of the `LDAP_Group_Filter_Group_Member_Format` setting here.